stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -310,6 +310,13 @@ function Base.copy(tB::Transpose{<:Any,<:Bidiagonal})
     return Bidiagonal(map(x -> copy.(transpose.(x)), (B.dv, B.ev))..., B.uplo == 'U' ? :L : :U)
 end
 
+@noinline function throw_zeroband_error(A)
+    zeroband = istriu(A) ? "lower" : "upper"
```

Comment:
IIRC then you had a recent PR where you avoided reading potentially `undef`ined values, and replaced this with `A.uplo == 'U'`?

Reply:
You're right!